.github/workflows/code_layout.yml

```diff
@@ -77,3 +77,19 @@ jobs:
           sudo apt install -y cppcheck
       - name: Run cppcheck test
         run: ./scripts/cppcheck.sh
+
+  translations:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Requirements
+        run: |
+          sudo apt update && sudo apt install qttools5-dev-tools
+          pip install wheel
+          pip install pygithub transifex-client
+
+      - name: "🌍 Push Translations"
+        if: ${{ github.event_name == 'push' }}
```

Comment:
Can we filter only pushes on master

Reply:
Are you sure? I copied the behaviour from the travis config, and it works on both `master` and `release-*` branches? https://github.com/opengisch/QField/blob/master/.travis.yml#L80

Reply:
We had this for a while, it's not sustainable (too many branches for translation are confusing to work with and get neglected).

I see much more potential for one master branch as translation sources. Most of the time these will match the sources in the latest release-* source. And as a backup plan we could merge locally stored translations with fetched master translations.

That said, I'm not opposed to a change in behavior if we have a sustainable approach.

Reply:
I see the problem you are trying to adress. What about keeping the branches on transifex and periodically exporting the translations from `master` and importing them into `release-xxx` branches? 

Side note: I just realized that the Code Layout workflow is not the best host for "Push Translations" action, so I create new workflow "Translations".

Reply:
> I see the problem you are trying to adress. What about keeping the branches on transifex and periodically exporting the translations from master and importing them into release-xxx branches?

There is also the "problem" that it kills statistics (it will show many untranslated strings for outdated versions) and will distract new translators by being confusing and giving them too many choices to make before they even start translating.

> Side note: I just realized that the Code Layout workflow is not the best host for "Push Translations" action, so I create new workflow "Translations".

:+1:

Reply:
Okay, the new `translations.yml` workflow is triggered only on pushes on the `master` branch. I turned off the travis push translations.